src/app/app.component.html

```diff
@@ -741,6 +741,19 @@
           </div>
           }
           <!-- /right button bar -->
+
+          <!-- MOB Emergency Button -->
+          <div class="mob-emergency-button">
```

Comment:
The button should be hidden when map interaction is active.
Wrap it with `@if(!isInteracting) {...}`